### ISSUE TITLE
Using Conditional Formatting on columns with remapped display values is confusing

### ISSUE THREAD

**Issue by jcohensolal** [metabase/metabase#11458]
**Is your feature request related to a problem? Please describe.**
Conditional formatting is a crucial feature when presenting results in a table, to allow the reader to find more quickly the cells that require attention. Today, the feature only works on numeric columns. Ideally you'd want to be able to highlight cells with critical text values for example.

**Describe the solution you'd like**
To be able to use non-numeric characters when you select a "is equal to" rule.

**Describe alternatives you've considered**
A possible alternative could be to use numeric-equivalent values for text values, for example transform "critical" to 1, "needs attention" to 2, etc. The problem is, you don't always have that possibility.

**How important is this feature to you?**
It would be a really welcome addition, as at this stage presenting big tables with text is not really an option without this feature IMHO.

**Comment by flamber** [comment]
Hi @juliencohensolal 
I might be misunderstanding, what you're looking for, but you can use strings too (tested on 0.33.6):
![image](https://user-images.githubusercontent.com/1447303/70343618-5ce74500-181d-11ea-9caa-f58bcb129972.png)

**Comment by juliencohensolal** [comment]
Hi @flamber,

it sure does look exactly like what I'm asking for, so now I'm confused. I'm using 0.33.4 and cannot enter non-numeric characters in that field, but I've browsed through the release notes of subsequent releases and didn't find anything related to this issue. Also originally it was only supposed to work on numeric features only (see [https://github.com/metabase/metabase/issues/7122](https://github.com/metabase/metabase/issues/7122)).

I guess it can be closed then?

**Comment by flamber** [comment]
@juliencohensolal 
I really cannot remember if this has been numeric-only, but this is how it looks on 0.33.4 w/ Firefox:
![image](https://user-images.githubusercontent.com/1447303/70454647-63b9c600-1a70-11ea-8dc7-d6ff25f07365.png)
So yes, please close this issue :wink: :+1:

**Comment by mazameli** [comment]
@juliencohensolal Is it possible that the column you're trying to use this on is a foreign key ID column that has been remapped to display the values of a column from the linked table? Another way to ask this is: what is the underlying data type of the column you're trying to do this on?

**Comment by bobatsar** [comment]
I tried to format based on a bool column and it didn't show up.

**Comment by flamber** [comment]
@bobatsar You should be able to select `boolean` columns, if you set Field Type as "Number" (Admin > Data Model > (database) > (table) > (column) :gear:), and using "is less than or equal to 0" for `false` and "is greater than or equal to 1" for `true`. I only did a quick test with MySQL, so I don't know if your database type might be handled differently.
Not a great experience, but a workaround. This feels more like a Completeness issue - specially if @juliencohensolal is using remapped display values, since it expects the ID instead of the shown display value, which is not intuitive either.

**Comment by mazameli** [comment]
Yeah, based on @juliencohensolal's comments, my hunch is the column they're trying to use is using remapped display values, which is why it's not possible to type non-numeric values into the conditional field. I'm going to retitle this issue and recategorize it as such, but please correct me if I'm wrong @juliencohensolal .

**Comment by juliencohensolal** [comment]
Sorry for not getting back to you earlier, we spun up a 0.33.6 version just to make sure. To make a long story short, I cannot reproduce the issue on our 0.33.6 version, while I still have it on the 0.33.4 version.

On the off-chance that there might be some underlying issue still around, I'll give some more details on this weird behaviour encountered on my 0.33.4 instance. I have a field "Day of Week" of "Category" type, taking "Monday", "Tuesday", ..., "Sunday" as possible values. When trying to create conditional formatting on this field, I can only enter numeric characters, and, extremely weirdly, the character "e". This is not a remapped foreign key. Screenshots below.

![mb1](https://user-images.githubusercontent.com/15109258/70895201-cb32c100-1fee-11ea-9188-9834c9c8a05c.JPG)
![mb2](https://user-images.githubusercontent.com/15109258/70895217-cff77500-1fee-11ea-8360-4dbdb782f502.JPG)

**Comment by flamber** [comment]
@juliencohensolal 
What is the actual database column type of "Day of Week" ?
What is the "Display values" in Admin > Data Model > (database) > (table) > Day of Week :gear: ?

**Comment by ranquild** [comment]
The problem seems to be that the FE formats numbers as day of week names in the table but it's not clear that the user needs to use numbers for conditional formatting.

**Comment by cdeweyx** [comment]
Closing as a duplicate of https://github.com/metabase/metabase/issues/9238
